build/build-image/Dockerfile

```diff
@@ -23,6 +23,9 @@ ENV GLIDE_VERSION=v0.12.3 \
 RUN curl -sSL https://github.com/Masterminds/glide/releases/download/$GLIDE_VERSION/glide-$GLIDE_VERSION-linux-amd64.tar.gz \
     | tar -vxz -C /usr/local/bin --strip=1
 
+RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.0-linux-amd64.tar.gz \
+    | tar -vxz -C /usr/local/bin --strip=1 etcd-v3.1.0-linux-amd64/etcd
+
```

Comment:
> it needs to be available during the integration tests, yes? Not the build?

True, but that doesn't mean it doesn't belong in the "build image." The thing we keep calling the "build image" is really a containerized _development environment_. Building is just one piece of that. Testing is another.

I have no problem at all with this addition.

If anything here is wrong, it's not introduced by this PR, and that is that it was a misnomer to call our containerized development environment "scbuildimage," because that name is simply too narrowly scoped for what it actually does.